generators/generator-base.js

```diff
@@ -472,6 +472,7 @@ module.exports = class extends PrivateBase {
             { name: 'Greek', value: 'el' },
             { name: 'Hindi', value: 'hi' },
             { name: 'Hungarian', value: 'hu' },
+            { name: 'Bahasa Indonesia', value: 'id' },
```

Comment:
Yes but in our intenal file we use inky english name for language and in english it is Indonesian. I checked Wikipedia